U/URCU/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "URCU"
+version = v"0.13.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://git.lttng.org/userspace-rcu.git", "c19d74445b6ed2e6840038afc2f974e3e098334b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/userspace-rcu/
+./bootstrap 
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
```

Comment:
Ok, it looks like Windows isn't supported at all, so I guess we can skip it
```suggestion
platforms = filter!(!Sys.iswindows, supported_platforms(; experimental=true))
```